include/tlog/errs.h

```diff
@@ -141,4 +141,68 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+/**
+ * Push an error to the perrs stack, with message as a constant string
+ * and then go to cleanup.
+ *
+ * @param fmt   The message string to copy and put into the pushed error.
+ *              Cannot be NULL.
+ */
+#define TLOG_ERRS_RAISES(_fmt) \
+     do {                                       \
+         tlog_errs_pushs(perrs, _fmt);          \
+         goto cleanup;                          \
+     } while (0)
+
+/**
+ * Push two errors to the perrs stack, with message being a global return code
+ * description, and with message as a constant string.
+ * Update the local grc variable by assigning _grc to grc,
```

Comment:
The macro doesn't care if "grc" is local or global. It should also be quoted. So something like "Assign _grc argument to "grc"" would work better.